index.js

```diff
@@ -0,0 +1,43 @@
+import express from 'express';
+import cors from 'cors';
+import morgan from 'morgan';
+import dotenv from 'dotenv';
+import chalk from 'chalk';
+import swaggerDoc from 'swagger-ui-express';
+import { swaggerControllers } from './controllers/index.js';
+
+dotenv.config();
+
+const app = express();
+
+const { PORT = 3000 } = process.env;
+
+app.use(morgan('short'));
+
+app.use(cors());
+
+app.use(express.json());
+
+app.use('/docs', swaggerDoc.serve, swaggerDoc.setup(swaggerControllers));
+
+app.use((_req, res) => {
+  res.status(404).json({
+    status: 'error',
+    message: 'Something went wrong...',
```

Comment:
here we can say that we don't have such endpoint, that's okay :)
we can even prepare 404 page and send it, if you wish

Reply:
We're going to do it. I've already prepared a special cool gif for the 404 page.